### PR TITLE
FSP approved list of app to clone

### PR DESCRIPTION
**Pull request by siascone**
Please read through the list and double check the Features/Bonuses. We didn't have listed features/bonuses for some of apps on our approved list so I selected ones that focused on the core of the apps.

### CODE REVIEW

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+- Shopping site
+- Tumblr
+- Facebook
+- Splitwise
+- Stack Overflow
+
+## Apps requiring Web-sockets:
+- Slack
+- Discord
+- Poll everywhere
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:29]:
```suggestion
- Poll Everywhere
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+
+## Other: 
+You may request a site of your choice but it will need to be approved. For best chance of approval make sure your site has the following:
+  - New account creation, login/signup
+  - At leaset one complte CRUD feature (preferably 2)
+  - Need for at least one addtional api:
+    - AWS (media storage)
+    - Google Maps
+    - Web-sockets
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:39]:
```suggestion
    - WebSockets
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+## Apps requiring Web-sockets:
+- Slack
+- Discord
+- Poll everywhere
+
+
+## Other: 
+You may request a site of your choice but it will need to be approved. For best chance of approval make sure your site has the following:
+  - New account creation, login/signup
+  - At leaset one complte CRUD feature (preferably 2)
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:35]:
```suggestion
  - At least one complete CRUD feature (preferably 2)
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+- 500px
+- Etsy
+- Amazon
+- Eventbrite
+- Flickr
+- Goodreads
+- Medium
+- LinkedIn
+- Pinterest
+- Shopping site
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:20]: Limit this list to specific apps and maybe make a note elsewhere about the wide variety of e-commerce applications available to clone

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
+# a/A Approved list of Apps
+
+## Apps requiring AWS:
+- All Trails - _Google Maps required_
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:4]:
```suggestion
- AllTrails - _Google Maps required_
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+- AirBnB - _Google Maps required_
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:5]:
```suggestion
- Airbnb - _Google Maps required_
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+- HipCamp - _Google Maps required_
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:6]:
```suggestion
- Hipcamp - _Google Maps required_
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+- Yelp - _Google Maps required_
+- Open Table - _Google Maps required_
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:8]:
```suggestion
- OpenTable - _Google Maps required_
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+- Streeteasy - _Google Maps required_
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:9]:
```suggestion
- StreetEasy - _Google Maps required_
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+
+## All Trails
+- Feature 1: Trails
+- Feature 2: Parks - _Google Maps_
+- Feature 3: Reviews/Ratings - _CRUD_
+- Feature 4: Search 
+- Bonus: Favorites 
+- Bonus: Completed trails
+
+## AirBnB
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:54]:
```suggestion
## Airbnb
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+- Feature 1: Listings - _CRUD_
+- Feature 2: Reservations - _CRUD_
+- Feature 3: Locations - _Google Maps_
+- Feature 4: Reviews - _CRUD_
+- Bonus: Experiences
+- Bonus: User/Host Porfiles
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:60]:
```suggestion
- Bonus: User/Host profiles
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+
+## HipCamp
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:62]:
```suggestion
## Hipcamp 
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+## Yelp 
+- Feature 1: Business Page
+- Feature 2: Search/Filters 
+- Feature 3: Reviews/Ratings - _CRUD_
+- Feature 4: Map - _Google Maps_
+- Bonus: Mark reviews funny, cool, useful etc.
+- Bonus: Profile
+- Bonus: Friends
+
+## Open Table
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:79]:
```suggestion
## OpenTable 
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 14]
```diff
@@ -0,0 +1,222 @@
...
+- Feature 2: Favorites - _CRUD_
+- Feature 3: Map - _Google Maps_
+- Feature 4: Search
+- Bonus: Ask a question
+- Bonus: Messages
+
+## 500px
+- Feature 1: Post Pictures - _CRUD_
+- Feature 2: Follows - _CRUD_
+- Feature 3: Home Feed
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:107]:
```suggestion
- Feature 3: Home feed
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 15]
```diff
@@ -0,0 +1,222 @@
...
+- Feature 4: Profile Page - _CRUD_
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:108]:
```suggestion
- Feature 4: Profile page - _CRUD_
```

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 16]
```diff
@@ -0,0 +1,222 @@
...
+- Feature 2: Shopping Cart - _CRUD_
+- Feature 3: Reviews/Ratings - _CRUD_
+- Feature 4: Search
+- Bonus: Categories
+- Bonus: Lists
+
+## Eventbrite
+- Feature 1: Events - _CRUD_
+- Feature 2: Registration/Tickets - _CRUD_
+- Feature 3: Categores
```
Comment (darreneid) [inline full_stack_projects/aa_approved_full_stack_projects.md:132]:
```suggestion
- Feature 3: Categories
```

File: full_stack_projects/mvp_guidelines.md [hunk 17]
```diff
@@ -0,0 +1,27 @@
+# MVP Guidelines
+
+Full-stack projects will be evaluated against the following "Minimal Viable Product" features. All 7 items must be complete in order for a full-stack project to be considered finished.
+
+Make sure outline your MVP features using the following structure when writing your Full-Stack Project Proposal. "Feature 1" through "Feature 4" should be replaced with your app's four key features. Required features for clones from our list of recommendations can be found below. For clones not included on the list, please check in with your TA mentor to clarify the app's key features.
```
Comment (darreneid) [inline full_stack_projects/mvp_guidelines.md:5]:
```suggestion
Make sure you outline your MVP features using the following structure when writing your Full-Stack Project Proposal. "Feature 1" through "Feature 4" should be replaced with your app's four key features. Required features for clones from our list of recommendations can be found below. For clones not included on the list, please check in with your TA mentor to clarify the app's key features.
```

**Review by darreneid — COMMENTED** [review]
Starting pointing out a few inconsistent styling things.
- some of the company/app names need to be adjusted (camelCase vs. space, etc.)
- capitalization of multi word lines in recommended features
- user/host vs. user / host

Also recommend using a specific example of an e-commerce site and then making a note that the same MVPs can generally be applied to a wide variety of apps.

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 18]
```diff
@@ -0,0 +1,222 @@
...
+- Feature 2: Feed
+- Feature 3: Follows - _CRUD_
+- Feature 4: Likes - _CRUD_
+- Bonus: Reblog
+- Bonus: User show page (blog)
+
+## Facebook
+- Feature 1: Profiles - _CRUD_
+- Feature 2: Friending - _CRUD_
+- Feature 3: Comments/Posting on each others' walls - _CRUD_
```
Comment (mwmadsen67) [inline full_stack_projects/aa_approved_full_stack_projects.md:197]: For the sake of makign sure people's projects have content, it's typically recommended to do Posts before Friends.

File: full_stack_projects/mvp_guidelines.md (hunk 17 above)
Comment (mwmadsen67) [inline full_stack_projects/mvp_guidelines.md:5]: Make sure to outline

**Review by mwmadsen67 — CHANGES_REQUESTED** [review]
left a few comments

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 19]
```diff
@@ -0,0 +1,222 @@
...
+    - Google Maps
+    - Web-sockets
+    - etc.
+
+---
+
+# Required Features for Approved Apps
+
+## All Trails
+- Feature 1: Trails
```
[the remark this answers is not in the document]
Comment (siascone) [reply full_stack_projects/aa_approved_full_stack_projects.md:47]: Parks have individualized trail indexes like all of the trails for Yosemite would be listed listed on the Yosemite National Park page. Trails would be the individual show page for one of the trails in the park.

File: full_stack_projects/aa_approved_full_stack_projects.md (hunk 4 above)
Comment (siascone) [reply full_stack_projects/aa_approved_full_stack_projects.md:20]: Updating in the next push

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 20]
```diff
@@ -0,0 +1,222 @@
...
+
+## Flickr
+- Feature 1: Photos - _CRUD_
+- Feature 2: Albums - _CRUD_
+- Feature 3: Comments - _CRUD_
+- Feature 4: Tags - _CRUD_
+- Bonus: Favorites
+- Bonus: Follows
+
+## Goodreads
```
[the remark this answers is not in the document]
Comment (siascone) [reply full_stack_projects/aa_approved_full_stack_projects.md:145]: I have seen a solid goodreads clone but I can see where you are coming from. While I like the simpleness of the app I don't like the look of the book form, it's very messy. While I have seen an avid user of the sight really enjoy working on this project I am happy to defer to your judgment and remove it from the list. We can approve upon request as needed.

File: full_stack_projects/aa_approved_full_stack_projects.md [hunk 21]
```diff
@@ -0,0 +1,222 @@
...
+- Feature 1: Profile - _CRUD_
+- Feature 2: Connections - _CRUD_
+- Feature 3: Posts - _CRUD_
+- Feature 4: Feed
+- Bonus: Notifications
+- Bonus: Messaging
+
+## Pinterest
+- Feature 1: Profile - _CRUD_
+- Feature 2: Boards and Pins - _CRUD_
```
[the remark this answers is not in the document]
Comment (siascone) [reply full_stack_projects/aa_approved_full_stack_projects.md:171]: Updating in next push

File: full_stack_projects/aa_approved_full_stack_projects.md (hunk 18 above)
Comment (siascone) [reply full_stack_projects/aa_approved_full_stack_projects.md:197]: Updated in next push

**Comment by siascone** [comment]
Suggested reviews made. Please reevaluate.

**Review by darreneid — APPROVED** [review]

File: full_stack_projects/aa_approved_full_stack_projects.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (siascone) [reply full_stack_projects/aa_approved_full_stack_projects.md]: updated

**Review by mwmadsen67 — APPROVED** [review]
looks good!

**Pull request merged**